tests/data/create_tables.sql

```diff
@@ -151,3 +151,10 @@ CREATE TABLE rest.gwascat_data (
   first_author TEXT,
   study TEXT
 );
+
+CREATE VIEW rest.recommended AS
```

Comment:
For my own reference: does this also represent the exact sql that would be used to migrate an *existing* database/deployment? Are there other conventions in this codebase for how we handle deployment of changes (migrations as separate files, etc)?

Reply:
DB changes are in the statgen/locuszoom-db repository. There's a branch tracking changes from these API changes here: https://github.com/statgen/locuszoom-db/tree/feature/recommended-ids. Since the changes are usually so small, I apply them manually to each database (dev or production) when it is time. 

There's no way to do automatic migrations with the existing system. Might be worth doing for API v2 someday. Maybe there are database change management tools that can diff/migrate automatically. Rewriting to use sqlalchemy/alembic is probably more work than it's worth at this point (before v2, anyway.)

Reply:
Thanks! I'll edit this into the PR description, as "how to deploy this" is fantastic and super useful information.

(Even if you expect to deploy the code to prod yourself, sometimes people have long-running dev environments and they need to find the updates required to go from time x to time y. Tagging that information is a little nicety)


_Not suggesting for this PR_, but sometimes it's useful to share strategies for a goal across other projects:

In the absence of autogenerated migrations, the next lower step of reproducibility on deployment is to script the migration steps as a SQL file that can be read in directly via the CLI.

Here's an approach we use on GfG. It's not perfect by a long stretch, and some operations, esp table changes, usually can't be done in a transaction.... but by capturing the exact sequence of commands in a tool and GUI agnostic way, it's been a helpful tool to ensure that dev environments stay in sync and operate according to the same rules.
https://github.com/statgen/genesforgood/blob/master/sql/migrations/research/061_surveys_purpose_life.sql

Reply:
Sounds good, trying that out here: 

https://github.com/statgen/locuszoom-db/blob/8a96cd8de5691767c530d2825d1d116ecc9ed960/db/migrations/0001_recommended_ids.sql

Is there a convention for denoting which migrations to apply to go from dev -> production? Or maybe from one DB "version" to the next?

Reply:
One thing we sometimes use is a special table called "migrations" to track what is applied. So in the gfg example above, the last command in every file is a migration name.

In ORMs, this table can be used to automatically find and apply all relevant migrations. In a change-database-by-hand system, it's usually more about keeping a record. (Somewhere to look if you think the dev and prod environments have drifted too much)